models/public/cocosnet/ade20k_ref_test.txt

```diff
@@ -0,0 +1,2000 @@
+ADE_val_00000001.jpg,ADE_train_00013948.jpg
```

Comment:
what the purpose of this file in model folder?

Reply:
@alexmasterdi  please remove and just describe how user should prepare data list for running demo

Reply:
This file is needed for accuracy check. I add it in model folder by analogy `vocab.txt`. This file don't use for demo and I can remove it, but how can I link accuracy checker to this file (example `vocab.txt` located in omz-validation-datasets)?